src/crypto/utils/api.py

```diff
@@ -16,6 +16,6 @@ def __init__(self, limit: int = 5000) -> None:
         }
 
     def request(self, url: str) -> dict:
-        response = requests.get(url, headers=self._headers, params=self._parameters)
+        response = requests.get(url, headers=self.headers, params=self.parameters)
```

Comment:
I was wondering if I would add a raise for status, but last time I did not:
https://github.com/PyBites-Open-Source/search/blob/main/src/pybites_search/base.py#L40